lib/rules/require-explicit-slots.js

```diff
@@ -34,9 +57,9 @@ module.exports = {
     if (!documentFragment) {
       return {}
     }
-    const scripts = documentFragment.children.filter(
-      (element) => utils.isVElement(element) && element.name === 'script'
-    )
+    const scripts = documentFragment.children
+      .filter(utils.isVElement)
+      .filter((element) => element.name === 'script')
```

Comment:
Yes, otherwise its type is `(VElement | VExpressionContainer | VText)[]`. Not sure how to directly implement the type assertion like TS `(element): element is VElement`, so just use `filter` twice.